Makefile

```diff
@@ -649,6 +649,11 @@ fmt-go:
 	cd server/log-collector && \
 		make fmt
 
+.PHONY: vale-docs
+vale-docs: ## Run vale check for docs and sorts ignore.txt file
+	vale docs
+	@sort .github/styles/MLRun/ignore.txt -o .github/styles/MLRun/ignore.txt
```

Comment:
What should I install in order for this to work?
If a Python dependency - add to `dev-requirements.txt`, otherwise document in `CONTRIBUTING.md`.